RayLin.md

```diff
@@ -1,14 +1,34 @@
-## 11/02 - 11/06
-
-- [Settings]
-	- [Fixed] [Bug 808141] Wi-Fi shows "offline" whereas it should be "Not connected"
-	- [Fixed] [Bug 1220532] the accessible-app-icon aria string is incorrect 
-	-[Keyboard]
-	- [WIP] [Bug 1219674] Implement \*test\_email\_keyboard.py\* as an integration test in JavaScript 
-	- [WIP] [Bug 1219675] Implement \*test\_keyboard.py\* as an integration test in JavaScript 
-	- [WIP] [Bug 1219676] Implement \*test\_keyboard\_bug\_1073870.py\* as an integration test in JavaScript 
-	- [WIP] [Bug 1219677] Implement \*test\_keyboard\_predictive\_key.py\* as an integration test in JavaScript 
-	- [Reviewing] [Bug 1219678] Implement \*test\_number\_keyboard.py\* as an integration test in JavaScript 
-
-- Study
-	- Firefox OS Developer Guide 
\ No newline at end of file
+# This Week
+
+- \[Keyboard\]
+	- Bug 1219674 Implement \*test\_email\_keyboard.py\* as an integration test in JavaScript 
+      - RESOLVE FIXED
+	- Bug 1219675 Implement \*test\_keyboard.py\* as an integration test in JavaScript 
+      - Should covered by others test suites eventually. 
+	- Bug 1219676 Implement \*test\_keyboard\_bug\_1073870.py\* as an integration test in JavaScript 
+      - Covered in Bug 1219677
+	- Bug 1219677 Implement \*test\_keyboard\_predictive\_key.py\* as an integration test in JavaScript 
+      - RESOLVE FIXED
+      - Quickfix typo
+	- Bug 1219678 Implement \*test\_number\_keyboard.py\* as an integration test in JavaScript 
+      - RESOLVE FIXED
+	- Bug 1216432 Keys on Emoji swipe panel does not layout correctly in landscape mode
+      - WIP
+      - Test scroll-snap and work smoother than javascript's appoarch
+      - Need to discuss with UX about scroll bar and indicator
```

Comment:
I showed UX the differences between two approaches, and they asked me to demo it on Flame for performance issue. I borrowed a flame device and re-flash it this morning, and I'll discuss with UX later.  It seems that they prefer JS approach if no obvious performance issues.